packages/contracts/contracts/voting/admin/Admin.sol

```diff
@@ -44,9 +44,11 @@ contract Admin is PluginCloneable, Proposal {
     /// @notice Creates and executes a new proposal.
     /// @param _metadata The metadata of the proposal.
     /// @param _actions The actions to be executed.
+    /// @param _allowFailureMap Allows proposal to succeed even if an action reverts. Uses bitmap representation. If the bit at index `x` is 1, the tx succeeds even if the action at `x` failed. Passing 0 will be treated as atomic execution.
     function executeProposal(
         bytes calldata _metadata,
-        IDAO.Action[] calldata _actions
+        IDAO.Action[] calldata _actions,
+        uint256 _allowFailureMap
```

Comment:
Does it make sense that the admin plugin needs this? If execution will be immediate, it might be cheaper to save one parameter and have the admin see that execution will fail before submitting it. The admin can split actions into different proposals with low overhead, unlike any governance plugin.

Happy to hear you view on this, but feel free to resolve if you recommend adding this here.

Reply:
It's still needed. Imagine, admin wants to execute, but is not sure whether or not 2nd action fails or not, but he doesn't want the overal tx to fail.  So uses bitmap param.

Really doesn't increase gas that much.

Reply:
The main goal of the admin plugin is to gain speed and save gas, compared to the other ones.

If the gas savings are light, then feel free to resolve the issue, but otherwise I would recommend to save one parameter and make execution atomic as it was.